.changeset/funky-functions-follow.md

```diff
@@ -0,0 +1,19 @@
+---
+"@biomejs/biome": patch
+---
+
+Type inference can now infer the return types of functions and methods without
```

Comment:
It seems we don't infer the return type when we have multiple `return` (based on tests).

If that's the case, we should reword the changeset to make it more clear. Users might misunderstood that

If it isn't the case, we should create a test for that (and maybe a snippet in the changeset too)